js.webpack.js

```diff
@@ -23,39 +22,41 @@ const entry = {
 };
 
 module.exports = (env) => {
-  const doHot = env.development && !env.watch_js;
```

Comment:
Need to check how this works with Ruby server and enable hot module replacement.